src/test/java/core/basesyntax/MyLinkedListTest.java

```diff
@@ -0,0 +1,112 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class MyLinkedListTest {
+
+    private static MyLinkedListInterface<String> myLinkedList = new MyLinkedList<>();
+    private static List<String> linkedList = new LinkedList<>();
+
+    private static final List<String> DEFAULT_LIST = new LinkedList<>(
+            Arrays.asList("First", "Second", "Third", "Fourth", "Fifth", "Sixth"));
+
+    @Test
+    public void test1Add() {
+        myLinkedList.add("First");
+        myLinkedList.add("Second");
+        myLinkedList.add("Third");
+        linkedList.add("First");
+        linkedList.add("Second");
+        linkedList.add("Third");
+        Assert.assertEquals(linkedList.toString(), myLinkedList.toString());
```

Comment:
@DemPaul You have overridden toString method in your solution. This is a bad idea. You shouldn't add any implementation to the _MyLinkedList_. Other students should do this. So this Assert is not useful. I guess comparing with _LinkedList_ in tests is a bad idea in the whole. You should take in a mind how _LinkedList_ works and expect the same behavior from _MyLinkedList_. Please refactor all tests where you are using toString comparing